src/components/WalletDropdown/MiniPortfolio/Pools/index.tsx

```diff
@@ -118,7 +118,13 @@ function PositionListItem({ positionInfo }: { positionInfo: PositionInfo }) {
       events={[BrowserEvent.onClick]}
       name={SharedEventName.ELEMENT_CLICKED}
       element={InterfaceElementName.MINI_PORTFOLIO_POOLS_ROW}
-      properties={{ chain_id: chainId, pool_token_0: pool.token0, pool_token_1: pool.token1 }}
+      properties={{
+        chain_id: chainId,
+        pool_token_0_symbol: pool.token0.symbol,
+        pool_token_1_symbol: pool.token1.symbol,
+        pool_token_0_address: pool.token0.address,
+        pool_token_1_address: pool.token1.address,
+      }}
```

Comment:
should this be wrapped in `useMemo`?

Reply:
https://github.com/Uniswap/analytics/blob/main/src/analytics/TraceEvent.tsx#L21

Reply:
ty for pointing this out!!